opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp

```diff
@@ -60,13 +60,21 @@ namespace Opm {
         };
 
         template<class E>
-        void logException(const E& exception, bool verbose)
+        void logException(const E& exception, bool verbose, const double newTimeStep = 0)
```

Comment:
I think this function should not be responsible for logging the timestep chopping. Also the search for and use of particular substrings is fragile, it will break if the messages are changed. Changing the function to use std::string instead of std::ostringstream is good though. The part starting with line 71 (found = message.find(...)) should be removed.